### PR TITLE
Performance e2e Tests

### PR DESCRIPTION
**Pull request by yanirq**
Performance end to end tests added: 
- Check Tuning profiles were applied
- Check feature set for latency is applied
- Check Pre boot tuning
- Check systcl values for realtime and network latency

### CODE REVIEW

File: functests/performance/performance.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+			checkFileExistence(workerRTNodes, perfRTKernelPatchScript)
+		})
+	})
+
+	Context("FeatureGate - FeatureSet configuration", func() {
+		It("FeatureGates with LatencySensitive should exist", func() {
+			fg, err := testclient.Client.FeatureGates().Get(components.FeatureGateLatencySensetiveName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			lsStr := string(ocv1.LatencySensitive)
+			By("Checking whether FetureSet is configured as " + lsStr)
```
Comment (davidvossel) [inline functests/performance/performance.go]: FetureSet/FeatureSet

**Comment by yanirq** [comment]
> tests look good.
> 
> Are we going to include any tests that exercise reserved cpu and isolated cpu related logic?

@davidvossel  Yes im looking into that, but that will probably follow in a different PR

**Pull request merged**